shopping_mall/backend/main.py

```
@@ -1,4 +1,24 @@
+    pass
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(levelname)-8s %(name)s: %(message)s",
+    stream=sys.stdout,
+    force=True,
+)
 
 if __name__ == "__main__":
-    uvicorn.run("app.main:app", host="0.0.0.0", port=4000, reload=True)
+    uvicorn.run("app.main:app", host="0.0.0.0", port=4000, reload=True, log_level="info")
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)